src/main/java/org/jenkinsci/test/acceptance/po/TopLevelItem.java

```diff
@@ -58,8 +58,7 @@ public TopLevelItem(PageObject context, URL url, String name) {
     }
 
     /**
-     * Renames the job. Opens the configuration section, sets the name and saves the form. Finally the rename is
-     * confirmed.
+     * Changes the description. Opens the configuration section, sets the description and saves the form.
```

Comment:
There is particular convention for setting form elements. Someone prefer public `Control`s, I prefer verbless methods, etc. I am fine with this.